src/database/models/SettingsModel.ts

```diff
@@ -1,15 +1,31 @@
 import {model, Schema} from 'mongoose';
 
+interface SettingsI {
+  _id: string, // Guild ID
+  prefix: string,
+  timezone: string,
+  verificationEnabled: boolean,
+  pinsEnabled: boolean,
+  quotesEnabled: boolean,
+  birthdayAnnouncementsEnabled: boolean,
+  morningAnnouncementsEnabled: boolean,
+  dueDatesEnabled: boolean,
+  emailDomain: string,
+  verifiedRole: string,
```

Comment:
Forgetting admin role

Reply:
Chose to check user permissions for that functionality instead.